reddit/settings.py

```diff
@@ -13,12 +13,12 @@
 from pathlib import Path
 
 from decouple import config
+from django.conf.urls import handler403
```

Comment:
I am pretty sure this can be removed, and then you will be able to remove #noqa from last line. Please check, and if possible, remove.

Reply:
Solved. All this code was unnecessary and the 403.html was in wrong place.

Reply:
So 403 is working?